README.markdown

```diff
@@ -7539,6 +7540,23 @@ This API was first introduced in the `v0.8.0` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.timer.every
```

Comment:
@doujiang24 Will you please make it a method in a separate Lua module `ngx.timer` in `lua-resty-core`? We do want to bloat the existing global `ngx.` giant table with any new API functions any more. It adds more GC overhead for every OR application.